controllers/login.js

```diff
@@ -0,0 +1,25 @@
+const ModelUser = require('../models/user')
+const Guid = require('guid')
+
+module.exports.login = async function login (req, res) {
+  try {
+    const { email, password } = req.body
+    const User = await ModelUser.findOne({email: email})
```

Comment:
I think we should not use capital letter with local variables, we must const 
`const user = await ModelUser.findOne({email: email})`

Reply:
Ok, got it, thanks